tests/samplers_tests/test_nsgaii.py

```diff
@@ -146,6 +148,151 @@ def constraints_func(trial: FrozenTrial) -> Sequence[float]:
         assert trial.system_attrs[_CONSTRAINTS_KEY] == (trial.number,)
 
 
+@pytest.mark.parametrize("dir1", [StudyDirection.MINIMIZE, StudyDirection.MAXIMIZE])
+@pytest.mark.parametrize("dir2", [StudyDirection.MINIMIZE, StudyDirection.MAXIMIZE])
+@pytest.mark.parametrize(
+    "constraints_list",
+    [
+        [[]],  # empty constraint
+        [[-float("inf")], [-1], [0]],  # 1d constraints
+        [
+            [c1, c2] for c1 in [-float("inf"), -1, 0] for c2 in [-float("inf"), -1, 0]
+        ],  # 2d-constraints
+    ],
+)
+def test_constrained_dominates_feasible_vs_feasible(
+    dir1: StudyDirection, dir2: StudyDirection, constraints_list: List[List[float]]
+) -> None:
+    directions = [dir1, dir2]
```

Comment:
Make it consistent with the above suggestion.
```suggestion
    direction1: StudyDirection, direction2: StudyDirection, constraints_list: List[List[float]]
) -> None:
    directions = [direction1, direction2]
```